hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/ExchangeRateManager.java

```diff
@@ -100,9 +104,19 @@ public void init(@NonNull final HederaState state, @NonNull final Bytes bytes) {
     /**
      * Updates the exchange rate information. MUST BE CALLED on the handle thread!
      *
-     * @param bytes The protobuf encoded {@link ExchangeRateSet}.
+     * @param bytes   The protobuf encoded {@link ExchangeRateSet}.
+     * @param payerId   The payer of the transaction that triggered this update.
      */
-    public void update(@NonNull final Bytes bytes) {
+    public void updateViaTransaction(@NonNull final Bytes bytes, @NonNull AccountID payerId) {
```

Comment:
Do we need to rename this method? Just asking because there are other "special file managers" that use the "update" name, IIRC, and maybe consistency would be good here.

Reply:
Any suggestions?
@tinker-michaelj

Reply:
just to update the public method